modules/src/core/ics04_channel/error.rs

```diff
@@ -324,6 +324,22 @@ define_error! {
                              e.port_channel_id.1)
             },
 
+        ProcessedTimeNotFound
+            { client_id: ClientId }
+            | e | {
+                format_args!(
+                    "Processed time for the client {0} not found",
+                    e.client_id)
+            },
+
+        ProcessedHeightNotFound
+            { client_id: ClientId }
+            | e | {
```

Comment:
Same nit as above: it would be good to capture the specific update_height in the error type.